client/src/pages/FeedPage.vue

```diff
@@ -19,8 +19,10 @@
               <q-icon name="filter_alt" />
             </template>
           </q-select>
-          <div class="col-4 col-md-3">
-            <q-btn dense class="full-width">Dismiss All</q-btn>
+          <div v-if="hasUnreadNotifications" class="col-4 col-md-3">
```

Comment:
If this will use the mark all read mutation this v-if needs to not be there.

Reply:
While making this change to the interface, I developed the opinion that the Dismiss All button appearing when there were no notifications to be dismissed felt clunky and made the interface appear more cluttered than necessary. 

If you disagree, then I will revert this behavior so that the Dismiss All button persists regardless of circumstances.